gateway-js/src/index.ts

```diff
@@ -903,6 +905,7 @@ export class ApolloGateway implements GraphQLService {
             serviceMap,
             requestContext,
             operationContext,
+            this.supergraphSchema!,
```

Comment:
Note to reviewer: I know this `!` is a tad "smelly", but this method does plenty of this earlier, with `this.apiSchema!` and `this.queryPlanner!` in particular, which are assigned in the same places than `this.supergraphSchema`, and I didn't exactly wanted to rework the entire lifecycle of the gateway in this patch.